Tests/Find-CTlsCertificate.Tests.ps1

```diff
@@ -150,29 +150,35 @@ Describe 'Find-CTlsCertificate' {
         GivenCertificate -For 'does not match hostname'
         GivenCertificate -For 'also does not match hostname'
         WhenFindingTlsCertificate 'example.com' -ErrorAction SilentlyContinue
-        ThenNoCertificateFound 
+        ThenNoCertificateFound
     }
 
     It 'should not find a certificate when no private key exists' {
         GivenCertificate -For 'noprivatekey.com' -WithNoPrivateKey
         WhenFindingTlsCertificate 'noprivatekey.com' -ErrorAction SilentlyContinue
-        ThenNoCertificateFound 
+        ThenNoCertificateFound
     }
 
     It 'should find the certificate when subject alternate name matches' {
-        GivenCertificate -For $machineName -WithDnsNames ('fake.net', 'fake2.net')
+        GivenCertificate -For $env:COMPUTERNAME -WithDnsNames ('fake.net', 'fake2.net')

```

Comment:
Use `[Environment]::MachineName` for cross-platform compatability. Not sure Linux/macOS have an `env:COMPUTERNAME` env var.

Reply:
Oops, good point.